babel.cfg

```diff
@@ -0,0 +1,5 @@
+[python: **.py]
+silent=False
+
+[jinja2: **.jinja]
+silent=False
```

Comment:
What does this config do?

Reply:
Now documented in 1c3ecbe07ea902df036cd3a28967f49e06ca410f.  :-)